packages/manager/apps/web/client/app/hosting/cdn/shared/add-edit-cache-rule/hosting-shared-add-edit-cache-rule.html

```diff
@@ -29,8 +29,29 @@
                     type="text"
                     data-ng-disabled="$ctrl.rule"
                     data-ng-model="$ctrl.ruleModel.name"
+                    ng-change="$ctrl.isValidRuleName($ctrl.ruleModel)"
                     required
                 />
+                <span
+                    class="help-block"
```

Comment:
this doesn't look like a ui-kit helper, don't we have ui-kit classes to display `oui-field` help text ?